docs/build.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+
+import importlib
+import logging
```

Comment:
While logging is used throughout the project, logs aren't shown when using `tox`. We may want to specify the log level:
```python
logging.basicConfig(level=logging.INFO)
```